src/styles/ui-kit-buttons.module.scss

```diff
@@ -0,0 +1,71 @@
+@import '@styles/colors.scss';
+
+.container {
+    font-family: Wix Madefor Text;
+    width: 284px;
+    padding: 16px;
+}
+
+.uikit {
+    font-size: 10px;
+    font-weight: 700;
+    line-height: 12.6px;
+    letter-spacing: 0.1em;
+}
+
+.coreComponents {
+    font-size: 10px;
+    letter-spacing: 0.1em;
+}
+.active {
+    color: var(--black);
+    font-weight: 600;
+}
+
+.hrSolid {
+    height: 1px;
+    width: 250px;
+    background-color: #000;
+}
```

Comment:
It exists exactly also in `ui-kit-typography.module.scss`. Maybe we can put it in "commons" or something shared like that? 
Not sure if this suggestion is true to how the project is built.

Same- `hrLight` and any other that exists also in the other style file.

Reply:
will solve in another pr :)